app/models/local_authority.rb

```diff
@@ -73,4 +75,8 @@ def self.from_my_society_import_row(data)
       end_date: data["end-date"]
     )
   end
+
+  def self.for_postcode(postcode)
+    joins(:local_authority_postcodes).where(local_authority_postcodes: {postcode: LocalAuthorityPostcode.normalize_postcode(postcode)}).first
```

Comment:
I think if you do something like this you shouldn't need to call `normalize_postcode` as it'll be done automatically by `normalizes`.

```rb
joins(:local_authority_postcodes).merge(LocationAuthorityPostcode.where(postcode:))
```